anomalib/data/mvtec.py

```diff
@@ -39,9 +39,9 @@
 from torch.utils.data.dataset import Dataset
 from torchvision.datasets.folder import VisionDataset
 
-from anomalib.data.transforms import PreProcessor
-from anomalib.data.utils import read_image
-from anomalib.utils.download_progress_bar import DownloadProgressBar
+from anomalib.data.utils.download_progress_bar import DownloadProgressBar
+from anomalib.pre_processing import PreProcessor
+from anomalib.utils.image import read_image
```

Comment:
Would this be located in data itself? Is there any similar example in other repos?